# ML-02 — Research Question and Provisional Lane

[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ronAa21/starternotebook/blob/main/work/notebooks/w01_research_question.ipynb)

This skeleton is yours to fill. Work the sections **in order** — each one has a one-line hint. Simple words, honest numbers.

> Working with an AI assistant? Tell it to read `skills/README.md` first and load the one skill this assignment names on its card.

## 1. My lane (or freestyle) and why

*Name your lane — or say 'freestyle' and describe your own question. One short paragraph: why this one?*

I chose lane 2: Refresh / Content Opportunity Scoring. Because it matches well with my experience building data-driven recommendations in machine-learning

## 2. The question: decision, action, cost of a wrong call

*What decision does your work improve? Who acts on it? What does a wrong recommendation cost?*

Decision: Which pages should be prioritized first for content review.

Action: The team reviews the highest priority page first, then decide whether to refresh, expand, or monitor them.

Cost of a Wrong Call: If the recommendation is wrong, the team might spend time reviewing low-impact pages while missing pages that have greater potential in increasing engagement

## 3. Quick look at the data (2-3 real numbers)

*Load the starter CSV below and show 2-3 real numbers that make your lane look worth the next 7 weeks.*

In [28]:

import pandas as pd

df = pd.read_csv(
    "starternotebook/data/raw/content_refresh_anonymized.csv"
)

print("Total pages:", len(df))
print("Average CTR:", df["ctr"].mean())
print("Median Position:", df["avg_position"].median())
print(df["trend_direction"].value_counts())

Total pages: 30000
Average CTR: 0.5107333333333334
Median Position: 10.8
trend_direction
down      16262
stable     5962
up         4388
new        2236
flat       1152
Name: count, dtype: int64


The csv file contains a total of 30000 data. The average for CTR is 0.51, and the median is 10.8. The data contains 16262 that're trending downward meaning there are a lot of data that may benefit from reviewing, refreshing, or monitoring, which makes the lane 2 a problem worth problem worth solving

## 4. Careful words: what I can and can't claim

*Write what your work will be able to say (observed, directional, decision-support) — and what it never will (causal proof, 'predicting Google').*

This capstone project will identify pages worth reviewing, refreshing, or monitoriing base on the data-driven results. But I can't claim that refreshing a page will definitely improve the performance, the traffic nor the engagements. The recommendations should be treated as decision-support tools for content teams rather than the one deciding for future outcomes



```
# This is formatted as code
```

## Self-check

Before you submit, confirm each line honestly:

- [ ] Every section above is filled — markdown thinking AND the code that backs it
- [ ] The notebook runs top to bottom with no errors (Runtime → Run all)
- [ ] No client names, URLs, or private queries anywhere
- [ ] My claims use careful words: observed, measured, directional, decision-support
- [ ] Committed to my repo under `work/notebooks/` — then submit your repo URL on the card. Done.